tests/contrib/flask/test_flask_appsec.py

```diff
@@ -253,3 +255,76 @@ def test_flask_body_xml_empty_logs_warning(self):
             self._aux_appsec_prepare_tracer()
             self.client.post("/", data="", content_type="application/xml")
             assert "Failed to parse werkzeug request body" in self._caplog.text
+
+    def test_request_ipblock_nomatch_200(self):
```

Comment:
same as for Django, I think snapshot tests would be good for these to verify only expected code paths are called/spans are created.

good way to ensure that if someone refactors the Flask integration that they don't accidentally move the blocking to the end of the request.

Reply:
Done.